tutorials/physics/using_character_body_2d.rst

```diff
@@ -439,30 +408,30 @@ And the code for the Bullet:
     public partial class Bullet : CharacterBody2D
     {
         public int Speed = 750;
-        private Vector2 _velocity = new Vector2();
 
-        public void Start(Vector2 pos, float dir)
+        public void Start(Vector2 _position, float _direction)
         {
-            Rotation = dir;
-            Position = pos;
+            Rotation = _direction;
+            Position = _position;
             _velocity = new Vector2(speed, 0).Rotated(Rotation);
         }
 
-        public override void _PhysicsProcess(float delta)
+        public override void _PhysicsProcess(double delta)
         {
             var collision = MoveAndCollide(_velocity * delta);
             if (collision != null)
             {
-                _velocity = _velocity.Bounce(collision.Normal);
-                if (collision.Collider.HasMethod("Hit"))
+                _velocity = _velocity.Bounce(collision.GetNormal());
+                if (collision.GetCollider().HasMethod("Hit"))
```

Comment:
What do you think about caching the result of `collision.GetCollider()` to avoid making 2 calls? I'm not sure how much we should focus on performance in tutorials.

Reply:
In a basic tutorial like this, I don't think it should be that much of a concern.